qiskit/providers/models/backendconfiguration.py

```diff
@@ -497,6 +498,14 @@ def __init__(self,
         self.meas_kernels = meas_kernels
         self.discriminators = discriminators
         self.hamiltonian = hamiltonian
+        if hamiltonian is not None:
+            self.hamiltonian = dict(hamiltonian)
+            self.hamiltonian['vars'] = {
+                k: v * 1e9 if isinstance(v, numbers.Number) else v
```

Comment:
Is there any instance where this is not a number? I believe the schema enforces that this is always a float.